keylime/tpm/tpm2_objects.py

```diff
@@ -71,11 +72,15 @@ def _pack_in_tpm2b(val: bytes) -> bytes:
 
 
 # The hash functions used by TPM
-HASH_FUNCS = {
-    TPM_ALG_SHA1: hashes.__dict__.get("SHA1"),
-    TPM_ALG_SHA256: hashes.__dict__.get("SHA256"),
-    TPM_ALG_SHA384: hashes.__dict__.get("SHA384"),
-    TPM_ALG_SHA512: hashes.__dict__.get("SHA512"),
+HASH_FUNCS: Dict[int, hashes.HashAlgorithm] = {
+    TPM_ALG_SHA1: hashes.SHA1(),
+    TPM_ALG_SHA256: hashes.SHA256(),
+    TPM_ALG_SHA384: hashes.SHA384(),
+    TPM_ALG_SHA512: hashes.SHA512(),
+}
+
+SYMCIPHER_FUNCS = {
+    TPM_ALG_AES: algorithms.AES,
```

Comment:
The user will have to instantiate this one with the actual key material, so we cannot instantiate it here..